giraffe/src/utils/formatStatValue.ts

```diff
@@ -35,14 +35,20 @@ export const formatStatValue = (
   digits = Math.min(digits, MAX_DECIMAL_PLACES)
 
   if (isNumber(value)) {
-    const roundedValue = Number(value).toFixed(digits)
-    const endsWithZero = /\.[1-9]{0,}0{1,}$/
-
-    localeFormattedValue = endsWithZero.test(roundedValue)
-      ? roundedValue
-      : Number(roundedValue).toLocaleString(undefined, {
-          maximumFractionDigits: MAX_DECIMAL_PLACES,
-        })
+    const roundedValueAndDecimal = Number(value)
```

Comment:
minor suggestion: use [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Examples) to make more self documenting code:
```ts
    const [wholeNumber, fractionalNumber] = Number(value).toFixed(digits).split('.')
```

then you can do something like

```ts
if (fractionalNumber) {
  ...
}
```

Reply:
Excellent suggestion. Done.